src/main/java/redis/clients/jedis/BinaryJedis.java

```diff
@@ -3469,19 +3470,18 @@ public Long lastsave() {
    * is switched off without the lost of any data. This is not guaranteed if the client uses simply
    * {@link #save() SAVE} and then {@link #quit() QUIT} because other clients may alter the DB data
    * between the two commands.
-   * @return Status code reply on error. On success nothing is returned since the server quits and
-   *         the connection is closed.
+   * @throws JedisException with the status code reply on error. On success nothing is thrown since
+   *         the server quits and the connection is closed.
    */
   @Override
-  public String shutdown() {
+  public void shutdown() throws JedisException {
```

Comment:
@sazzad16 shouldn't we deprecate this method first?

Reply:
@gkorland Why deprecate? We're not removing it.

Reply:
@gkorland Please check #2545